cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -39,6 +56,9 @@ do
   do
     expirationDateValue=$(echo ${json_output} | jq --arg cl "${cluster}" -r -S '.Vpcs[] | select (.Tags[]? | (.Key == $cl and .Value == "owned")) | .Tags[] | select (.Key == "expirationDate") | .Value')
     handle_cluster "${cluster}" "${expirationDateValue}" "${r}"
+    echo "oc status"
+    oc status
```

Comment:
What will this print for us? status of which namespace, for every cluster?

Reply:
Removed.
I was expecting to use this to confirm that oc-login was set up ok.